int8 id
sensor_msgs/Image faceImage
geometry_msgs/Pose pose
int32 reward
std_msgs/String prisonColor
std_msgs/Bool isPoster
